Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Armoury.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class Armoury : MonoBehaviour
+{
+    public Player player;
+    public PointsSystem currency;
+
+    IList<int> GetBuyableUnits; //Change to UnitType once Units are in
+    IList<int> GetBuyableCards;
+    bool b_IsCastlePieceBuyable;
+
+    IDictionary<int, int> UnitPrices { get; set; } //Change key to UnitType once Units are in
```

Comment:
Any reason why these are IDictionary rather than Dictionary? I think it would make sense in a function parameter but not so much as a local/public variable.